.github/workflows/vib-early-feedback.yaml

```
@@ -42,8 +42,9 @@ jobs:
             echo "::set-output name=result::fail"
           elif [[ "$num_charts_changed" -eq "1" ]]; then
             # Changes done in only one chart -> OK
-            echo "::set-output name=chart::${charts_dirs_changed}"
-            if [[ "$charts_dirs_changed" =~ "common" ]]; then
+            chart_name=$(echo "$charts_dirs_changed" | sed "s|bitnami/||g")
+            echo "::set-output name=chart::${chart_name}"
+            if [[ "$chart_name" =~ "common" ]]; then
```

I think we can change this condition to search for the exact match now that we are using the chart's name instead of its directory.
```suggestion
            if [[ "$chart_name" == "common" ]]; then
```

---

Done, thanks!